plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -614,7 +615,18 @@ private ConnectorTableMetadata doGetTableMetadata(ConnectorSession session, Sche
             throw new TrinoException(UNSUPPORTED_TABLE_TYPE, format("Not a Hive table '%s'", tableName));
         }
 
-        if (!translateHiveViews && isHiveOrPrestoView(table)) {
+        boolean isTrinoView = isPrestoView(table);
```

Comment:
It would be nice rename the method too

Reply:
agreed, but obviously out of scope for this PR, right?